app/assets/javascripts/hub.js

```diff
@@ -0,0 +1,10 @@
+$(document).on('ready',function(){
+	$('.location').click(function(){
+		$('.tags-search').css({"display":"none"});
+		$('.location-search').css({"display":""});
+	})
+	$('.tags').click(function(){
+		$('.tags-search').css({"display":"inline-block"});
+		$('.location-search').css({"display":"none"});
+	})
+})
```

Comment:
Think about consistency for indentation (across files) and semi-colon use. People fight about the latter. I like them but don't feel strongly about them - but do be consistent.